src/styles/gatsby-override.css

```
@@ -9,3 +9,5 @@ ic-typography label {
   font: inherit;
   letter-spacing: inherit;
 }
+
+a:visited { color: #6611cc !important; }
```

Please can you update it to use our visited link token?
```suggestion
a:visited { color: var(--ic-hyperlink-visited) !important; }
```